### PR TITLE
[vmware] add SRM recorded tests

### PR DESCRIPTION
**Pull request by ctaggart**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
vmware

**Comment by ctaggart** [comment]
Hey @zhoxing-ms, can you merge these test recordings? This has been open for a week.

File: src/vmware/azext_vmware/_help.py [hunk 1]
```diff
@@ -277,7 +277,7 @@
     short-summary: Create a Site Recovery Manager (SRM) addon for a private cloud.
     examples:
     - name: Create a Site Recovery Manager (SRM) addon.
-      text: az vmware addon srm create --resource-group MyResourceGroup --private-cloud MyPrivateCloud --license-key "41915178-A8FF-4A4D-B683-6D735AF5E3F5"
+      text: az vmware addon srm create --resource-group MyResourceGroup --private-cloud MyPrivateCloud --license-key "41915-178A8-FF4A4-DB683-6D735"
```
Comment (zhoxing-ms) [inline src/vmware/azext_vmware/_help.py:280]: May I ask why do we need to modify the `--license-key`?

Comment (ctaggart) [reply src/vmware/azext_vmware/_help.py:280]: It was the wrong format before. The test service now validates the format.

Comment (zhoxing-ms) [reply src/vmware/azext_vmware/_help.py:280]: OK

**Review by zhoxing-ms — APPROVED** [review]

**Pull request merged**